src/app/controller/CallingHomeControllerV2.php

```diff
@@ -0,0 +1,55 @@
+<?php
+namespace axonivy\update\controller;
+
+use Psr\Http\Message\ServerRequestInterface as Request;
+use Psr\Http\Message\ResponseInterface as Response;
+
+use axonivy\update\repository\ProductLogRepository;
+use axonivy\update\repository\ReleaseInfoRepository;
+use axonivy\update\model\ProductLogRecord;
+
+
+class CallingHomeControllerV2
+{
+    private ProductLogRepository $productLogRepo;
+    private ReleaseInfoRepository $releaseInfoRepo;
+
+    public function __construct(ProductLogRepository $productLogRepo, ReleaseInfoRepository $releaseInfoRepo)
+    {
+        $this->productLogRepo = $productLogRepo;
+        $this->releaseInfoRepo = $releaseInfoRepo;
+    }
+
+    public function product(Request $request, Response $response)
+    {
+        $payload = json_decode($request->getBody());
+        $name = $payload->IvyProduct->name;
+        $version = $payload->IvyProduct->version;
```

Comment:
these to properties are the only required ones. 
although we don't really need them for our logstash consumption, it could simplify queries on the database records. In addition, it will create failures, if these properties are not present in the client payload. So basically it is also a minimal protection of user that send bad payload objects ... we won't store if these props are missing.